### ISSUE TITLE
Cannot read property 'fallbacks' of undefined

### ISSUE THREAD

**Issue by oliviertassinari** [cssinjs/jss#489]
## Reproduction

1. Navigate to http://cssinjs.org/repl
2. Use:
```js
return {
  tutu: undefined,
  font: {
    color: 'white',
  },
};
```

## Output

> Cannot read property 'fallbacks' of undefined

**Comment by oliviertassinari** [comment]
## Side note:

The behaviour changed with v7.

**Comment by kof** [comment]
sounds like a regression.

**Comment by kof** [comment]
Looking into it.

**Comment by kof** [comment]
fixed in published in 7.1.4

**Comment by oliviertassinari** [comment]
Thanks!
